server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -173,3 +212,17 @@ func isRootModified(root terraform.Root, modifiedFiles []string) (bool, error) {
 
 	return false, nil
 }
+
+func (r *Runner) partitionPRsByDateModified(now time.Time, prs []github.PullRequest) ([]github.PullRequest, []github.PullRequest) {
+	nDaysAgo := now.AddDate(0, 0, -ThroughputCutOffDays)
+	oldPRs, newPRs := []github.PullRequest{}, []github.PullRequest{}
+	for _, pr := range prs {
+		if pr.UpdatedAt.Before(nDaysAgo) {
```

Comment:
Reiterating another comment by saying this can just be done dynamically, not sure why you need to precompute this.